cfpb/cfgov-refresh
# Ask search and category pages

Displays and paginates results on Ask search and category pages

## Additions

- Spanish category pages
- Paginated results on category and search pages
- Side nav on English search page
- Option for round icon in `category-icon.html`
- Ability to rename `prev` and `next` buttons in `pagination.html` 

## Testing

- Check out branch
- Set up Ask
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- Run gulp
- Navigate to [English](http://localhost:8000/ask-cfpb/category-student-loans/) and [Spanish](http://localhost:8000/obtener-respuestas/categoria-pagar-la-universidad/) category pages and [English](http://localhost:8000/ask-cfpb/search/?q=credit) and [Spanish](http://localhost:8000/obtener-respuestas/buscar-es/?q=credito) search pages and check that question results are displayed and paginated

## Screenshots

![screencapture-localhost-8000-ask-cfpb-category-auto-loans-1494511459852](https://cloud.githubusercontent.com/assets/778171/25955152/44fe876c-361d-11e7-9e3c-8791b2553b87.png)
![screencapture-localhost-8000-ask-cfpb-search-1494511502334](https://cloud.githubusercontent.com/assets/778171/25955154/46f7b6ce-361d-11e7-92e2-9ea7c4d82fa0.png)
<img width="1245" alt="screen shot 2017-05-11 at 7 05 44 am" src="https://cloud.githubusercontent.com/assets/778171/25955161/4ac7c78a-361d-11e7-84ef-7b2ee8b889a4.png">
<img width="1245" alt="screen shot 2017-05-11 at 7 06 12 am" src="https://cloud.githubusercontent.com/assets/778171/25955162/4c6d0c26-361d-11e7-8151-44f377388780.png">

## Todos

- There's currently a failing test that I still need to fix

## Checklist
